packages/react-progress-bar/lib/react-progress-bar.js

```diff
@@ -30,13 +30,20 @@ export const ProgressBar = ({
 		</span>
 	);
 
+	const checkValue = (value) => {
+		if(null !== value) {
```

Comment:
@creador-dev, initially, we were not checking for a null value but an empty string. The primary purpose of the task is to fix the way we check if the `value` has an empty string. In this case, you created a function to check the `null` value.